MapifyEditor/Vanilla/VanillaObject.cs

```diff
@@ -5,5 +5,7 @@ namespace Mapify.Editor
     public class VanillaObject : MonoBehaviour
     {
         public VanillaAsset asset;
+        public bool keepChildren = true;
+        public Vector3 rotation = Vector3.zero;
```

Comment:
This should be renamed "rotationOffset" or something similar since it's not setting the exact rotation.